src/main/scala/com/lightbend/rp/sbtreactiveapp/App.scala

```diff
@@ -63,6 +63,7 @@ sealed trait App extends SbtReactiveAppKeys {
     // I will need to investigate why `akkaClusterBootstrapEnabled` is a task, not a setting.
     enableServiceDiscovery := false,
     akkaClusterBootstrapEndpointName := "akka-remote",
+    akkaClusterBootstrapManagementEndpointName := "akka-mgmt-http",
```

Comment:
There's a 15 character limit on K8s, hence this shortened string

Reply:
Added this to the validation card: https://github.com/lightbend/sbt-reactive-app/issues/13